src/GroupProject/Class/PaymentJson.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using GroupProject.DAL;
+
+namespace GroupProject.Class
+{
+    public class PaymentJson
+    {
```

Comment:
weird name since no data here is actually Json
but dont really have a suggestion for another name either